css/styles.css

```diff
@@ -89,3 +93,151 @@ a {
 .card:hover {
   box-shadow: 0 0 1rem var(--tertiary-color);
 }
+
+/* Home Page Styling */
+
+.banner {
+  background: url(/assets/images/banner2.jpg);
+  background-size: cover;
+  height: 29rem;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  object-fit: contain;
+}
+
+.banner-content {
+  margin: auto;
+}
+
+.banner-title {
+  font-size: 3rem;
+  font-weight: 900;
```

Comment:
use helper class for font wait or define in root

Reply:
Will do that 
Thanks For Review